Propagators

DominicDirkx opened the pull request:
New Propagation beta code

magnific0 commented:
@DominicDirkx can you explain the reason for removing `Tudat/Astrodynamics/StateDerivativeModels`. I can see how there is overlap between the functionality of your code and these old files, however, there is some code (example applications) that depends on this. Moreover, I can imagine that user applications will be broken as well.

DominicDirkx commented:
@magnific0 I deleted the `Tudat/Astrodynamics/StateDerivativeModels` foilder, as all functionality that was included in it has now been subsumed into the Propagators folder. I thought it would be  less confusing to delete it. I have updated versions of the applications that I will create pull requests for once this is committed.

However, it might be a good idea to not delete the folder until all documentation etc. has been updated to the new structure.

magnific0 commented:
Hi @DominicDirkx 

My changes are all very minor. We already discussed brinigng back the old statederivative models (for now).

Based on this I had a few general remarks, but I'll open seperate issues for that.

The addition looks very good. I'm happy to see such big improvements in the usability of Tudat.
